### DISCUSSION TITLE
Open Code Search in a repository with keyboard shortcut

### DISCUSSION THREAD

**Discussion by saket** [community/community#12445] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey folks, I often find myself opening cs.github.com in repositories by manually typing `repo:user/repository search-term`. Is there a keyboard shortcut for automating this? I'm effectively asking for a shortcut that does the opposite of your ["," shortcut](https://github.com/github/feedback/discussions/10163#discussioncomment-1992829).

**Comment by colinwm** [comment, marked as answer]
Hey @saket, great suggestion! I've added a keyboard shortcut, Ctrl+Shift+F, which jumps to blackbird (only if you're in the technology preview). So e.g. if you're on an organization page or a repository page, or even reading a PR or an issue, you can search the corresponding org or repository. 

Let me know what you think!

**Reply by saket** [reply]
This is going to be so helpful, thank you!

**Comment by zakariaridouh** [comment]
Hi! Is there any way to have cs.github.com default to searching a particular repo? Some kind of user setting or even something like `http://cs.github.com/q=repo:user/repository+searchterm`? Thanks!
